vignettes/WorkflowInfo.Rmd

```diff
@@ -0,0 +1,127 @@
+---
+title: "Workflow Information"
+package: AnVIL
+output:
+  BiocStyle::html_document
+vignette: >
+  %\VignetteIndexEntry{Workflow Information}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8} 
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+    eval =  AnVIL::gcloud_exists(), collapse = TRUE, cache = TRUE
+)
+options(width=75)
+```
+
+# Run avworkflow_info()
+
+Load AnVIL package.
+
+```{r, message = FALSE, eval = TRUE, cache = FALSE}
+library(AnVIL)
+```
+
+## Multiple workflows in one job 
+
+When submissionId of the job is provided, workflow information can be 
+retrieved as below. 
+
+```{r, eval = FALSE}
+submissionId <- "9385fd75-4cb7-470f-9e07-1979e2c8f193"
+res1 <- avworkflow_info(submissionId)
+```
+
+Workflow information for sample submissionId 
+"9385fd75-4cb7-470f-9e07-1979e2c8f193" retrieved as above is stored 
+as res1.rds. 
+
+```{r, message=FALSE, warning=FALSE}
+res1 <- readRDS(system.file(package = "AnVIL", "data", "res1.rds"))
```

Comment:
Reconsider storing the results of these operations. The user should be able to run these examples on AnVIL.

Reply:
Will be in a new pull request. res1 and res2 are stored in /inst/data